docs/en/setup/README.md

```diff
@@ -25,6 +25,12 @@ You can go to their project repositories for additional info about guides and re
 
 - [SkyAPM GO2Sky](https://github.com/SkyAPM/go2sky). See GO2Sky project document for more details.
 
+## Browser agent
+
+- Activate the [browser-receiver](backend/backend-receivers.md#browser-receiver) following the document.
+
+- [Browser Js Client](https://github.com/apache/skywalking-client-js). See `skywalking-client-js` project document for more details.
```

Comment:
Move this to the 1st point. And change to 

[Browser JS Client](https://github.com/apache/skywalking-client-js). JS client reports the browser performance data to the backend. See `skywalking-client-js` project document for more details.

Reply:
done